docs/docs/admin/teamforcapella/full-setup/full-setup.md

```diff
@@ -0,0 +1,28 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+This tutorial will guide you through the full setup of a new TeamForCapella
+based project in the Collaboration Manager.
+
+1. Create a new project:
+   [Create a new project](../../../user/projects/create.md)
```

Comment:
Looks a little funny doubled like that, and other steps don't follow this convention. Probably cleaner as "[Create a new project] in the Collaboration Manager". Also most of the steps end with periods, but some are missing them (this one, 10, and 11 I believe)